draft-ietf-tls-dnssec-chain-extension-08.xml

```diff
@@ -605,47 +708,14 @@
   </section> <!-- trust anchor maintenance  -->
 
 
-  <section title="Mandating use of this extension" anchor="mandating">
-
-    <t>
-      Green field applications that are designed to always employ this
-      extension, could of course unconditionally mandate its use.
-    </t>
-
-    <t>
-      This protocol allows TLS servers to prove that they don't have a
-      signed TLSA record by returning a denial of existence chain.
-      However, as explained in <xref target="denial_of_existence"/>,
-      it does not require TLS servers to do so. In the absence of such
-      a proof, a TLS client misdirected to a server that has fraudulently
-      acquired a public CA issued certificate for the real server's name,
-      could be induced to establish a PKIX verified connection to the rogue
-      server that precluded DANE authentication. Application ecosystems
-      where all TLS servers are expected to implement this extension could
-      require such authenticated denial proofs to be delivered by TLS
-      servers that don't have signed TLSA records.
-    </t>
-
-    <t>
-      One possible way to address the threat of attackers that have
-      fraudulently obtained valid PKIX credentials, is to use current
-      PKIX defense mechanisms, such as checking Certificate Transparency
-      logs to detect certificate misissuance. This may be necessary anyway,
-      as TLS servers may support both DANE and PKIX authentication. Even
-      TLS servers that support only DANE may be interested in detecting
-      PKIX adversaries impersonating their service to DANE unaware TLS
-      clients.
-    </t>
-
-  </section>
-
-
   <section title="DANE and Traditional PKIX Interoperation" anchor="pkix-interop">
 
     <t>
       When DANE is being introduced incrementally into an existing
       PKIX environment, there may be scenarios in which DANE
       authentication for a server fails but PKIX succeeds, or vice versa.
+      Some consequences of such scenarios are further discussed in
+      <xref target="increasing-trust" />
       What happens here depends on TLS client policy. If DANE authentication
       fails, the client may decide to fallback to traditional PKIX
```

Comment:
FWIW "fallback" in the original text should be "fall back", the single word is a noun.